packages/spotlight/src/components/spotlight.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+
+import { useMemo, useRef, useState } from "react";
+import { ActionIcon, Center, Group, Kbd, Text } from "@mantine/core";
+import { Spotlight } from "@mantine/spotlight";
+import { IconSearch, IconX } from "@tabler/icons-react";
+
+import { useI18n } from "@homarr/translation/client";
+
+import type { inferSearchInteractionOptions } from "../lib/interaction";
+import { searchModes } from "../modes";
+import { searchEnginesChildrenOptions, searchEnginesSearchGroups } from "../modes/external/search-engines-search-group";
+import { selectAction, spotlightStore } from "../spotlight-store";
+import { SpotlightChildrenActions } from "./actions/children-actions";
+import { SpotlightActionGroups } from "./actions/groups/action-group";
+
+export const NewSpotlight = () => {
+  const [query, setQuery] = useState("");
+  const [mode, setMode] = useState("default");
```

Comment:
Maybe change to `help` for now and change later once we have search engines and something to show on the start page